lib/mjsonwp/mjsonwp.js

```diff
@@ -245,8 +245,8 @@ function buildHandler (app, method, path, spec, driver, isSessCmd) {
         driverRes = null;
       }
 
-      // assuming everything worked, set up the json response object
-      httpResBody.status = JSONWP_SUCCESS_STATUS_CODE;
+      // Response status should be the status set by the driver response.
+      httpResBody.status = driverRes.status;
```

Comment:
how about:

```js
httpResBody.status = _.isUndefined(driverRes.status) ? JSONWP_SUCCESS_STATUS_CODE :
                                                       driverRest.status;
```

to ensure that a poorly-constructed server we're proxying to doesn't cause a failure?